### DISCUSSION TITLE
Docker for testing Superset Failed in local

### DISCUSSION THREAD

**Discussion by Tana8M** [apache/superset#18262] [Q&A / Help]
A clear and concise description of what the bug is Docker

### Expected results
-I want to test superset dashboard.

### Actual results
-When I login page , that haven't anything in localhost:8088 (http://localhost:8088/superset/welcome)

#### Screenshots

![Screen Shot 2564-01-29 at 15 50 25](https://user-images.githubusercontent.com/53141452/106253601-4898af80-624a-11eb-959a-2f93dd668802.png)

**Comment by abhii-singh** [comment]
did you run `superset-init` ? it takes a while for all the assets to load 

Would make it easier if you can share some logs .

**Comment by SuzGori** [comment]
same issue.

superset-init done.

but same page.

woops.

superset-node is don't finished...

**Comment by acvander** [comment]
I experienced the same issue earlier. I followed the instructions in #9880 and increased the memory allocation in Docker to 8GB. 

The `superset-node` service does take a long time to complete installing the node modules. For me it took at least 20 minutes. You can follow the logs using ```docker-compose logs superset-node```. 

My logs looked like this while it was installing the modules
`
npm WARN using --force I sure hope you know what you are doing.
/usr/local/bin/webpack-cli -> /usr/local/lib/node_modules/webpack-cli/bin/cli.js
/usr/local/bin/webpack -> /usr/local/lib/node_modules/webpack/bin/webpack.js
 webpack-cli@4.4.0
 webpack@5.19.0
added 131 packages from 161 contributors in 20.531s
npm WARN using --force I sure hope you know what you are doing.
`

**Comment by yerrochdi** [comment]
same issue.
- superset_init not running 
superset_init            | ######################################################################
superset_init            | 
superset_init            | 
superset_init            | Init Step 4/4 [Complete] -- Loading examples
superset_init            | 
superset_init            | 
superset_init            | ######################################################################
superset_init            | 
superset_init exited with code 0

- superset_tests_worker not runing 
"Flask-Caching: CACHE_TYPE is set to null, "
ERROR:flask_appbuilder.security.sqla.manager:DB Creation and initialization failed: (psycopg2.OperationalError) FATAL:  database "test" does not exist

thank you for help

**Comment by SuzGori** [comment]
@yerrochdi 
test_worker is not important for me.
this is "test".
exit code 0 and work is important for me.

**Comment by murinmat** [comment]
did you guys follow https://superset.apache.org/docs/installation/installing-superset-using-docker-compose ? I have just tried it and got no error, all is running well on http://localhost:8088

I also tried: **$ docker run -p 8080:8080 -d apache/incubator-superset** which started superset container.
I then got into the container with **$ docker exec -it <container_id> /bin/bash**, ran **superset db upgrade** and **superset init**, created an admin account with **superset fab create-admin** and all is working in that part as well.

**Comment by vineetgupta02** [comment]
I followed this link and was able to get superset up and running with just one docker image 
https://hub.docker.com/r/apache/superset 

hope this helps.

**Comment by dongliang36k** [comment]
i have same wrong

**Comment by WilixLead** [comment]
Hi! I am new in superset, but when I try docs in https://hub.docker.com/r/apache/superset - got empty response on 8080 (as docs say).
When I change `docker run -d -p 8080:8080 --name superset apache/superset` to `docker run -d -p 8088:8088 --name superset apache/superset` - I got response on localhost:8088.
Looks like mistake in docker-hub page

**Comment by elewis787** [comment]
This still seems like an issue.

After running docker-compose up - the login screen and welcome page had no assets loading.  

I solved this by going into superset-frontend on the master branch and manually install the node modules. Then re-ran docker-compose up. You should see webpack building the assets/deps .

i.e 
```
[webpack.Progress] 55% building 13/19 entries 36000/36081 dependencies 8989/10856 modules
```
